### PR TITLE
Clarify that the password is a string and not a plain token as the username

### PR DESCRIPTION
**Pull request by Hunterness**
Also correct wrong index syntax...

### CODE REVIEW

**Comment by Hunterness** [comment]
[CREATE USER syntax / data types](https://trello.com/c/fuYcmsB4/2677-create-user-syntax-data-types)

**Comment by Hunterness** [comment]
- [ ] Have not built and looked at it
- [ ] Do we want to update 4.1 as well? (4.0?)

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc [hunk 1]
```diff
@@ -90,7 +90,7 @@ STATUS {ACTIVE \| SUSPENDED}
 
 | [source, cypher, role=noplay]
 ----
-ALTER CURRENT USER SET PASSWORD FROM original TO password
+ALTER CURRENT USER SET PASSWORD FROM 'original' TO 'password'
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc]:
```suggestion
ALTER CURRENT USER SET PASSWORD FROM 'password' TO 'new_password'
```

**Review by martin-neotech — COMMENTED** [review]
Small fix

**Comment by martin-neotech** [comment]
> * [ ]  Do we want to update 4.1 as well? (4.0?)

Yes please, it is such a tiny fix so lets fix it for 4.0 and 4.1 also. Thanks.

**Review by martin-neotech — APPROVED** [review]
Thanks

**Pull request merged**